app-connector-client/server/app.js

```diff
@@ -36,7 +35,21 @@ module.exports = function (varkesConfigPath = null, nodePortParam = null) {
     app.get("/", function (req, res) {
         res.render('index', { appName: varkesConfig.name })
     })
-
+    app.get("/logo", function (req, res) {
+        var ext = "image/svg+xml"
+        var img_path = path.resolve(__dirname, 'views/static/logo.svg')
+        if (varkesConfig.logo) {
+            img_path = varkesConfig.logo;
+            var tmp = img_path.split(".")
+            ext = "image/" + tmp[tmp.length - 1]
+            if (ext = "image/svg") {
```

Comment:
that is an assignment, not a comparison!

Reply:
removed this part